pkgs/development/python-modules/spectra/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, wheel
+, colormath
+}:
+
+buildPythonPackage rec {
+  pname = "spectra";
+  version = "0.0.11";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "jsvine";
+    repo = "spectra";
+    rev = "v${version}";
+    hash = "sha256-4A2TWTxYqckJ3DX5cd2KN3KXcmO/lQdXmOEnGi76RsA=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = [
+    colormath
+  ];
+
+  pythonImportsCheck = [ "spectra" ];
+
+  meta = with lib; {
```

Comment:
Same here:

```suggestion
  meta = {
```

Reply:
These are coming as a default from nix-init FYI. So if they're bad practice, maybe we should open up an issue there to remove them.